pycvodes/_config.py

```diff
@@ -7,6 +7,9 @@
 import tempfile
 import warnings
 
+# type_of_prec below needs to have an identical definition in ._types
```

Comment:
The _types file is a good idea.  I suppose if you wanted, type_of_prec could be moved entirely there so there's only one definition?

Reply:
I tried that in the second commit, but since setup.py uses "exec" (to avoid circular imports) _config cannot import anything from pycvodes (hence commit number 3)